src/backend/aazdev/commands.py

```diff
@@ -0,0 +1,18 @@
+# -----------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# -----------------------------------------------------------------------------
+
+from knack.commands import CommandGroup
+
+
+def load_command_table(self, _):
+    def operation_group(name):
+        return f"aazdev.operations.{name}#{{}}"
+
+    with CommandGroup(self, "configuration", operation_group("configuration")) as g:
```

Comment:
I think the word configuration is too long, we can use cfg for short.

```suggestion
    with CommandGroup(self, "cfg", operation_group("configuration")) as g:
```